x-pack/plugins/canvas/canvas_plugin_src/functions/common/alterColumn.ts

```diff
@@ -5,8 +5,15 @@
  */
 
 import { omit } from 'lodash';
+import { ContextFunctionSpec, Datatable, DatatableColumn, DatatableColumnType } from '../types';
 
-export function alterColumn() {
+interface Arguments {
+  column: string;
+  type: 'null' | 'boolean' | 'number' | 'string' | null;
```

Comment:
this seems to be duplicated, or close enough, to the one for `DatatableColumnType` of `types/common.ts`, maybe it could be extracted out